lib/chef/resource/aws_cache_cluster.rb

```diff
@@ -9,17 +12,59 @@ class Chef::Resource::AwsCacheCluster < Chef::Provisioning::AWSDriver::AWSResour
   # for information on possible values for each attribute. Values are passed
   # straight through to AWS, with the exception of security_groups, which
   # may contain a reference to a Chef aws_security_group resource.
+
+
+  # Cluster Name
+  #
+  # @param :cluster_name [String] unique name for a cluster
   attribute :cluster_name, kind_of: String, name_attribute: true
+
+  # Availability Zone
+  #
+  # @param :az_mode [String] availability zone of the cache cluster
   attribute :az_mode, kind_of: String
+
+  # Preferred Availability Zone
+  #
+  # @param :preferred_availability_zone [String] preferred availability zone of the cache cluster
   attribute :preferred_availability_zone, kind_of: String
+
+  # Preferred Availability Zones
+  #
+  # @param :preferred_availability_zones [String, Array] One or more preferred availability zones
   attribute :preferred_availability_zones,
             kind_of: [ String, Array ],
             coerce: proc { |v| [v].flatten }
+
+
+  # Number of Nodes
+  #
+  # @param :number_nodes [Integer] Number of nodes in the cache
   attribute :number_nodes, kind_of: Integer, default: 1
+
+  # Node type
+  #
+  # @param :node_type [String] AWS node type for each cache cluster node
   attribute :node_type, kind_of: String, required: true
+
+  # Engine
+  #
+  # @param :engine [String] Valid values are `memcached` or `redis`
   attribute :engine, kind_of: String, required: true
+
+  # Engine Version
+  #
+  # @param :engine_version [String] Valid values are `memcached` or `redis`
   attribute :engine_version, kind_of: String, required: true
+
+  # Subnet Group Name
+  #
+  # @param :subnet_group_name [String] Cache cluster subnet group
```

Comment:
I think its worth calling out that this should be a `aws_cache_subnet_group` and not just `aws_subnet`